analytical_engine/CMakeLists.txt

```diff
@@ -11,21 +11,22 @@ project(analytical_engine LANGUAGES C CXX VERSION ${GRAPHSCOPE_ANALYTICAL_VERSIO
 
 option(NETWORKX "networkx on?" ON)
 option(BUILD_TESTS "Build unit test" ON)
-option(ENABLE_JAVA_SDK "Build with support for java sdk" OFF)
+option(ENABLE_JAVA_SDK "Build with support for java sdk" ON)
 
 # Get system processor
 execute_process(COMMAND uname -m OUTPUT_VARIABLE SYSTEM_PROCESSOR)
 string(REGEX REPLACE "\n$" "" SYSTEM_PROCESSOR "${SYSTEM_PROCESSOR}")
 
-if (APPLE AND ENABLE_JAVA_SDK)
-    message(FATAL_ERROR "Java SDK is currently not supported on macos.")
+if (CMAKE_HOST_SYSTEM_NAME MATCHES "Darwin")
+    set(SYSTEM_PROCESSOR "arm64")
```

Comment:
macOS with Intel chip also need to be supported.

Reply:
macOS with x86 chip should be supported. Maybe I should delete this check, since it not necessary.